pennylane/devices/preprocess.py

```diff
@@ -151,22 +151,21 @@ def mid_circuit_measurements(
     tape: qml.tape.QuantumTape,
     device,
     mcm_config=MCMConfig(),
-    interface=None,
+    **kwargs,  # pylint: disable=unused-argument
```

Comment:
I will get rid of `**kwargs` after lightning is updated to stop using `interface` in the kwargs passed to the `mid_circuit_measurements` preprocessing transform